### ISSUE TITLE
Dispatchers taking a long time to start in VoiceConnections where one has already played

### ISSUE THREAD

**Issue by zapk** [hydrabolt/discord.js#2017]
**Please describe the problem you are having in as much detail as possible:**
I'm dynamically creating dispatchers in the same voice connection to play an ogg vorbis sound when necessary. There is never more than one dispatcher at the same time. However, I'm noticing a correlation between the number of times a dispatcher has played and the time it takes for new ones to actually start emitting sound.

**Include a reproducible code sample here, if possible:**
```js
_doNext() {
	console.log(`[Debug] _doNext`);
	this.voiceQueueRunning = false;

	if (!this.voiceQueue.length)
		return;

	let buffer = Buffer.concat(this.voiceQueue);
	if (!buffer)
		return;
		
	this.voiceQueue = [];

	const connection = this.voiceChannel.connection;
	if (!connection)
		return;

	this.voiceQueueRunning = true;

	const dispatcher = connection.playStream(new Readable({
		read() {
			this.push(buffer);
			buffer = null;
		}
	}));
	console.log(`[Debug] connection.playStream`);

	dispatcher.on('end', (reason) => {
		console.log(`[Debug] Ended, reason: ${reason}`);
		if (reason == 'Stream is not generating quickly enough.') {
			setTimeout(() => {
				this._doNext();
			}, 100);
		} else if (reason) {
			//bot-specific stuff
		}
	});

	dispatcher.on('error', console.error);
}
```

**Further details:**
"[Debug] connection.playStream" prints to the console the instant it should, but it can take upwards of 40 seconds before the bot starts sending voice data, starting at no delay and increasing over time.

- discord.js version: 11.2.1
- node.js version: 8.6.0
- Operating system: Ubuntu 16.04 xenial
- Priority this issue should have – please be realistic and elaborate if possible: low

- [ ] I found this issue while running code on a __user account__
- [ ] I have also tested the issue on latest master, commit hash:

**Comment by Dev-Yukine** [comment]
this issue is related to #1693 and you can find a temporary workaround in there also its already fixed on master with the commit #1745

**Comment by zapk** [comment]
Thanks. I should've thought to check closed issues…
